legisloop-backend/src/main/resources/application.properties

```diff
@@ -6,17 +6,17 @@ legiscan.base.url=https://api.legiscan.com
 springdoc.swagger-ui.path=/swagger-ui.html
 
 #API keys
-newsapi.api.key=
-legiscan.api.key=
+newsapi.api.key=${NEWSAPI_API_KEY}
+legiscan.api.key=${LEGISCAN_API_KEY}
 
 #Postgres
-spring.datasource.url=jdbc:postgresql://${DB_HOST:localhost}:5432/legisloop
-spring.datasource.username=postgres
-spring.datasource.password=postgrespassword
+spring.datasource.url=${SPRING_DATASOURCE_URL:postgresql://postgres_container:localhost}:5432/legisloop}
```

Comment:
Should this be `${SPRING_DATASOURCE_URL:postgresql}://${postgres_container:localhost}:5432/legisloop}`?

Reply:
Also, what about the `jdbc:` header?

Reply:
Is `postgres_container` supposed to be `postgres_db`?